h/schemas/pagination.py

```diff
@@ -1,16 +1,29 @@
+from dataclasses import dataclass
+
 from colander import Integer, Range, Schema, SchemaNode
 
+from h.schemas.util import validate_query_params
```

Comment:
A bit of generalization of pagination here to avoid duplicating the same in the new endponit.